app/pages/lab/workflows-container.jsx

```diff
@@ -60,13 +79,18 @@ export default class WorkflowsContainer extends React.Component {
     this.getWorkflowList();
   }
 
+  toggleReorder() {
+    this.setState((prevState) => { return { reorder: !prevState.reorder }; });
+  }
```

Comment:
should this call `getWorkflowList`? 

in NfN's case it works as expected initially, with 20 (of 100) workflows listed with pagination at the bottom, but once clicking the Reorder View button should it show all 100 workflows within the DragReorderable component?